timeline.js

```diff
@@ -0,0 +1,43 @@
+import { ActivityPubDB } from "./db.js";
+
+class ReaderTimeline extends HTMLElement {
+  constructor() {
+    super();
+    // Default outbox URLs to fetch and display in the timeline
+    this.outboxUrls = [
+      "ipns://staticpub.mauve.moe/outbox.jsonld",
+      "ipns://hypha.coop/outbox.jsonld",
+    ];
+  }
+
+  connectedCallback() {
+    this.initTimeline();
+  }
+
+  async initTimeline() {
+    const db = await ActivityPubDB.load();
```

Comment:
We should load the DB once somewhere. Maybe in a seperate script which exports the db?